SwanCustomEnvironments/swancustomenvironments/scripts/builders/python3.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
Can you name this script `venv` instead of `python3`?

Reply:
Yeah, but changing `python3-venv` to `venv-python3` is something that also has to be done on `options_from_form` (gitops)

Reply:
Ok, I think it makes sense though.